src/RFC-0131_Mining.md

```diff
@@ -95,16 +95,13 @@ good chance of being "commoditized" when ASICs are eventually manufactured. This
 
 ### The block distribution
 
-A 50/50 split in hash rate among algorithms minimises the chance of hash rate attacks. However,
-sufficient buy-in is required, especially with regard to merge mining RandomX with Monero. To make it worthwhile for a
-Monero pool operator to merge mine Tari, but still guard against hash rate attacks and to be inclusive of independent
-Tari supporters and enthusiasts, a 60/40 split is employed in favour of merge mining RandomX with Monero.
+A 50/50 split in hash rate among algorithms minimises the chance of hash rate attacks. In Tari, a 50/50 split is employed in favour of merge mining RandomX with Monero and Tari with Sha-3x respectively.
```

Comment:
I think we can remove this whole section. It was previously there to explain the 60/40 split

Reply:
Noted and removed.